### DISCUSSION TITLE
Help: Can't view 3d model in AR with Safari IOS

### DISCUSSION THREAD

**Discussion by carlosefin** [google/model-viewer#3470] [Q&A]
![IMG_123F9324E239-1](https://user-images.githubusercontent.com/37450858/168523439-25020878-b1df-49c3-adf9-0d0e1a8cb91a.jpeg)
A bunch of text is shown when I click the "view in AR" button.

**Comment by milesgreen** [comment]
That looks like the browser Brave. Which at this time I don't think supports viewing USDZ / AR QuickLook.

**Reply by elalish** [reply]
To be fair, we also shouldn't show the AR button in browsers that don't support AR. Can you confirm the browser and ideally paste in the `userAgent` string here?

**Comment by PGilbanks87** [comment]
Hi. I'm setting up a demo of model viewer and I'm having this same problem. Screenshot attached. This is using the chrome browser on an iOS device running 15.4.1. Screenshot from my iOS device attached. I have also had the same text appear in the Safari Browser.
![IMG_3948](https://user-images.githubusercontent.com/105852963/169263701-a69ab04c-17ea-4a0e-8ae7-9e8e906f9471.PNG)

**Reply by milesgreen** [reply]
For Chrome, see https://github.com/google/model-viewer/issues/3292

**Reply by PGilbanks87** [reply]
Thank you. Updated to iOS 15.5 and now its working.
